test/SILOptimizer/Inputs/linker_pass_input.swift

```diff
@@ -2,23 +2,26 @@
 @_silgen_name("unknown")
 public func unknown() -> ()
 
+@_inlineable
 public func doSomething() {
   unknown()
 }
 
-@_semantics("stdlib_binary_only")
 public func doSomething2() {
   unknown()
 }
 
 @inline(never)
-@_semantics("stdlib_binary_only")
 public func doSomething3<T>(_ a:T) {
   unknown()
 }
 
-struct A {}
+@_versioned struct A {
+  @_versioned init() {}
+}
+
 @inline(never)
+@_inlineable
```

Comment:
Wait, I thought `@inline(never)` and `@_inlineable` don't play well together, and already wanted to ask for a warning.

Reply:
`@_inlineable` is not about inlining, so they are actually orthogonal.

Reply:
But, @swiftix told me that `@inline(__always)` implies `@_inlineable`. I'm confused now...

Reply:
@moiseev IIRC, the current rules in the compiler work in such a way that  `@inline(__always)` and `@_transparent` implies @_inlineable. 

@slavapestov That's how it works today, or?